index.html

```diff
@@ -5132,6 +5132,12 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type MUST NOT be used to describe a verifiable credential with an <dfn>embedded proof</dfn>.
+  </p>
```

Comment:
```suggestion
```

@OR13,

My [objection](https://github.com/w3c/vc-data-model/pull/1014#discussion_r1102145924) was marked resolved without addressing my concern: https://github.com/w3c/vc-data-model/pull/1014#discussion_r1103268921

> I'm merging https://github.com/w3c/vc-data-model/pull/1014#discussion_r1102128119
> 
> based on the support I can see for it.

I see that some people support that statement. I object to it. We don't have consensus yet.

I currently think it's the wrong thing to do and I've asked for a technical basis for it and one or more clear [concrete examples](https://github.com/w3c/vc-data-model/pull/1014#discussion_r1102172924) -- otherwise I think it should be removed.

Reply:
> I see that some people support that statement. I object to it. We don't have consensus yet.

Agreed, but I raised this PR, and I can merge suggestions from WG members that I agree with.

You are welcome to raise a separate PR that implements the consensus you wish the WG to adopt.

Or request changes on this PR given it is heading a direction you do not agree with.

Reply:
Technically speaking, a W3C verifiable credential is also a W3C credential. It does not make sense to me to say that you can't tag a VC with an embedded proof with the media type `application/credential+ld+json`.

I've heard an argument that to do this would be similar to making a mistake like JOSE's `alg: none`, but this argument is not technically clear. I've also argued that perhaps the opposite is true:

1. If the `alg: none` argument is metaphorical, then I say that no one should depend on a media type to determine whether a credential is to be trusted. So if the statement banning using this media type with content with an embedded proof is meant to imply that a credential tagged with this media type can therefore be trusted -- that is bad security practice. To me, that bad practice would be metaphorically similar to `alg: none`.
2. If the `alg: none` argument refers to a literal problem, then I say that `alg: none` itself should be prohibited regardless of anything we do here; therefore, this should be an orthogonal concern.

I've heard an argument that using `application/credential+ld+json` "tells an implementer something important and to treat it differently". But this offers nothing specific nor does it surface any particular choices as to how things would be different if someone did happen to tag content with an embedded proof with that media type.

In fact, as an implementer, I would not expect to change any behavior whether an embedded proof was present or not -- other than potentially having the option to try and verify such a credential. But, if my software never tries that anyway (which would have to be the case if the statement were accepted), I do not expect to start rejecting credentials just because they have embedded proofs on them in these circumstances. That seems harmful and like I would be missing out on processing otherwise useful credentials.

It also seems like there may be use cases that are made more complex (having to handle two different media types when it's otherwise unnecessary) if embedded proofs are disallowed. As an example, a use case that comes to mind involves signing VCs with multiple proofs via proof sets. The first time a proof is attached to the credential, a caller would need to use media type 1, and the second time, media type 2. But existing implementations today that attach such proofs do not need to care about this, so this adds unnecessary complexity. Given my experience with what developers tend to do in these situations, I'd expect some of them to just start ignoring the prohibition statement, leading to interoperability problems. So, this prohibition statement needs to have real value, IMO.

Now, if this statement is not present, we don't have to worry about any of that. If we're going to have such a restriction it should be clearly articulated why -- both the benefits and the drawbacks, and a technical discussion should proceed from that followed by arrival at consensus. Otherwise, it's not worth including it at the moment and doing so could be harmful, IMO.

Reply:
@dlongley can you please request changes so it is clear that this PR cannot be merged without addressing / discussing them?

Reply:
> @dlongley can you please request changes so it is clear that this PR cannot be merged without addressing / discussing them?

You may have missed this -- I did that above as the first thing in this particular subthread: https://github.com/w3c/vc-data-model/pull/1014#pullrequestreview-1294114323

Reply:
@dlongley that is a suggestion. Under "add a review" there's a "request changes" button that is a clear "do not merge" signal

Reply:
![Screenshot 2023-02-10 at 3 32 21 PM](https://user-images.githubusercontent.com/7622243/218221871-d929cbc7-f62a-4988-9762-9fb913a388c8.jpg)

Reply:
I mean it's literally called "add a suggestion" - in every project I've worked on I haven't taken it to be a blocking request. only official reviews of which the options are [approve, comment, request changes]

<img width="223" alt="Screenshot 2023-02-10 at 4 10 56 PM" src="https://user-images.githubusercontent.com/7622243/218224874-3912c0a1-b865-4cc2-8a56-328e327c52de.png">

Reply:
@OR13, @decentralgabe,

:) ... as I said above, that's the first thing I did here. To be clear, I submitted a review where I chose "Request Changes" and included specific suggested changes (as can be seen above). I do appreciate all of the screen shots of what I did though! :P

I think what you may not be expecting is that only code owners / required reviewers get the red change icon and big red merge blocking warnings. It doesn't matter how hard I clicked the "Request Changes" button (and I did), it only shows a gray icon and treats my objection as coming from a mere peon worthy of dismissal. :)

Reply:
> it only shows a gray icon and treats my objection as coming from a mere peon worthy of dismissal

Editor hat on -- it's typically bad form for any Editor to dismiss change requests in a way that hides the fact that there continues to be disagreement on a particular line in a PR. 

@OR13 was acting in good faith in an attempt to make the PR easier to read, however, in dismissing the change request and the resulting debate, no one else coming into this PR can see that there was clearly disagreement on a particular line.

Additionally, as Editors, it's our job to ensure that if we are not to take suggestions from WG Members (and the public at large) that we have the consent of the individual that raised the change request in the first place... because if we don't have consent, the PR will just be blocked upon merge with a request to roll back the PR (causing extra and unnecessary churn).

As a general rule, as an Editor, I won't merge a PR that has any pending suggestions (note, NOT change requests) that have not explicitly been closed/resolved by the individual that raised them OR where there isn't clear agreement that they're fine w/ their change request not being merged.

When you resolve controversial change requests without the consent of the person that raised the change request, it almost always leads to further conflict in the PR.

We have been able to merge PRs for 10+ years w/o resorting to using the "Resolve conversation" button on threads containing disagreement. 

That said, different Editors have different PR management styles... and the WG empowers the Editors to manage PRs in the way they feel will lead to reaching consensus of the group.

> in every project I've worked on I haven't taken it to be a blocking request.

Open standards are not open source, incubation, or corporate projects, they're managed differently because the bar for reaching consensus is much higher at W3C (much to the dismay of a number of our WG members) :P.